### PR TITLE
Upgrade to use Gihub Actions, drop travis-ci, normalize our analysis …

### PR DESCRIPTION
**Pull request by schmidtw**
…tools, Dockerfiles and Makefiles.

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  range: 50..80
+  round: down
+  precision: 2
+
+ignore:
+  - "*_test.go"
+  - "vendor"
+
+fixes:
+  - "github.com/xmidt-org/scytale/::"
```
Comment (kcajmagic) [inline .codecov.yml:11]:
```suggestion
  - "github.com/xmidt-org/argus/::"
```

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+vendor
+scytale
```
Comment (kcajmagic) [inline .dockerignore:2]:
```suggestion
argus
```

**Review by kcajmagic — CHANGES_REQUESTED** [review]

**Review by kcajmagic — APPROVED** [review]

**Review by kristinaspring — APPROVED** [review]

File: .github/workflows/docker-release.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          if [ "${{ github.event_name }}" = "push" ]; then
+            TAGS="$TAGS,${DOCKER_IMAGE}:sha-${GITHUB_SHA::8}"
+          fi
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1` | tail -1)
+          if [ "${LATEST_TAG}" == "${VERSION}" ]; then
+            TAGS="$TAGS,${DOCKER_IMAGE}:latest"
+          fi
+          echo ::set-output name=version::${VERSION}
+          echo ::set-output name=tags::${TAGS}
+          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```
Comment (joe94) [inline .github/workflows/docker-release.yml:38]:
```suggestion
          echo ::set-output name=created::$(date -u '+%c')
```
To keep it consistent with our makefile buildtime

Comment (kristinaspring) [reply .github/workflows/docker-release.yml:38]: This time isn't used for building 👍

**Pull request merged**